package.json

```diff
@@ -73,7 +74,7 @@
   },
   "homepage": "https://github.com/join-monster/join-monster#readme",
   "peerDependencies": {
-    "graphql": "0.6 || 0.7 || 0.8 || 0.9 || 0.10 || 0.11 || 0.12 || 0.13"
+    "graphql": "^15.2.0"
```

Comment:
Does 15.0.0 work? If yes, would be nicer to devs if we did `^15.0.0`

Reply:
It would work for JS users but not for TypeScript users sadly -- the right base types for the `extensions` property didn't land until `15.2.0`. I have a slight preference to things just working out of the box, and I don't think `graphql` 15->15.2 has any breaking changes.